src/SelectPicker/test/SelectPickerSpec.js

```diff
@@ -400,4 +400,22 @@ describe('SelectPicker', () => {
       expect(getByTestId('content')).to.have.text('Not selected');
     });
   });
+  it('Should call onSearch when closed', async () => {
+    const handleSearch = sinon.spy();
+    const handleClose = sinon.spy();
+    let { container } = render(
+      <>
+        <button id="exit">exit</button>
+        <SelectPicker onClose={handleClose} defaultOpen onSearch={handleSearch} data={data} />
+      </>
+    );
+    const exit = container.querySelector('#exit');
+
+    // close select
+    fireEvent.mouseDown(exit, { bubbles: true });
+    await waitFor(() => assert.isTrue(handleClose.calledOnce));
+
+    assert.isTrue(handleSearch.calledOnce);
```

Comment:
Duplicated assertions. You could wrap multiple assertions within `waitFor`.

Reply:
Great writing style